frontend/packages/dev-console/src/plugin.tsx

```diff
@@ -471,6 +471,16 @@ const plugin: Plugin<ConsumedExtensions> = [
         )).default,
     },
   },
+  {
+    type: 'Page/Route',
+    properties: {
+      path: ['/helm-releases/all-namespaces/release/:name', '/helm-releases/ns/:ns/release/:name'],
```

Comment:
I don't think this is valid to have a helm release name with no namespace?
For an other pages we show the project list page.